src/garage/torch/algos/loss_function_utils.py

```diff
@@ -1,10 +1,25 @@
-"""loss function utilities."""
+"""Differentiable utils typically used when computing loss functions."""
 import torch
 import torch.nn.functional as F
 
 
 def compute_advantages(discount, gae_lambda, max_len, baselines, rewards):
-    """Calculate advantages."""
+    """
+    Calculate advantages.
+
+    Advantages are a discounted cumulative sum.
+
+    Args:
+        discount (float): Discount.
+        gae_lambda (float): Lambda used for generalized advantage estimation.
+        max_len (int): Maximum length of a single rollout.
+        baselines (torch.Tensor): The baseline values.
+        rewards (torch.Tensor): The rewards.
```

Comment:
You can omit articles, e.g. "The". This could read "Rewards" or "Time series of rewards"

Reply:
fixed